models.py

```diff
@@ -0,0 +1,29 @@
+# This encodes the different models
+
+def m1(expe,alphaG = .1,alphaN=.1,alphaC=.1,betaG=1,betaN=1):
```

Comment:
Use a more descriptive name for the function.

I would use either 'exp' or 'experiment' for the first param -- abbreviating after a vowel is weird.

Unless you have a good reason to deviate, use standard python variable naming conventions: lower_case_with_underscores, so alpha_g, alpha_n, etc.

Space after commas.
No space around the '=' when specifying default parameters.
(together, that makes it easy to visually split all the function args)